indie_label_svelte/src/SubmitReportDialog.svelte

```diff
@@ -3,22 +3,35 @@
     import Button, { Label } from "@smui/button";
     import Textfield from "@smui/textfield";
     import CircularProgress from '@smui/circular-progress';
+    import Checkbox from '@smui/checkbox';
 
     export let open;
     export let cur_user;
     export let all_reports;
     let name = "";
     let email = "";
+    let which_reports_to_submit = [];
```

Comment:
Minor: maybe we can add a comment that this is an array of the same length as `all_reports` consisting of booleans that indicate whether the report should be submitted or not, since it's a bit more of a complicated setup if someone wasn't familiar with the code!